.pre-commit-config.yaml

```diff
@@ -35,6 +32,11 @@ repos:
     -   id: trailing-whitespace
         types: [python]
 
+-   repo: https://gitlab.com/pycqa/flake8
```

Comment:
Why is this - side-effect-ly - added to this branch and not added via separate branch?

Reply:
of course you are correct. It was just that I couldn't commit anything because pre-commit kept failing. What do you suppose? I could cherry pick the commit and open another PR